src/manifolds/ProductManifold.jl

```diff
@@ -234,8 +227,11 @@ end
 function det_local_metric(
```

Comment:
Here the same seems to hold as for power
(1) `M.manifold is the product manifold, so it should be `M.manifold.manifolds`
(2) due to `ProductMetric` being the default metric this seems to be transparent – in the most recent commit there is a commented out line that fails for comparison (bottom of product manifold tests).

Reply:
Yes, I think both need the exact same fix.